src/components/VerticalListMenu/VerticalListMenu.jsx

```diff
@@ -0,0 +1,209 @@
+		 * Appended to the component-specific class names set on the root element.
+		 */
+		className: string,
+
+		/**
+		 * Passed through to the root element.
+		 */
+		style: object,
+
+		/**
+		 * Indicates which of the `VerticalListMenu.Item` children are currently
+		 * selected. You can also put the `isSelected` prop directly on the
+		 * `VerticalListMenu.Item`s if you wish.
+		 */
+		selectedIndices: arrayOf(number),
+
+		/**
+		 * Indicates which of the `VerticalListMenu.Item` children are currently
+		 * expanded. You can also put the `isExpanded` prop directly on the
+		 * `VerticalListMenu.Item`s if you wish.
+		 */
+		expandedIndices: arrayOf(number),
+
+		/**
+		 * Callback fired when the user selects a `VerticalListMenu.Item`.
+		 *
+		 * Signature: `(index, { event, props }) => {}`
+		 */
+		onSelect: func,
+
+		/**
+		 * Callback fired when the user expands or collapses an expandable `VerticalListMenu.Item`.
+		 *
+		 * Signature: `(index, { event, props }) => {}`
+		 */
+		onToggle: func,
+	},
+
+	getDefaultProps() {
+		return {
+			onSelect: _.noop,
+			onToggle: _.noop,
+			expandedIndices: [],
+			selectedIndices: [],
+		};
+	},
+
+	render() {
+		const {
+			children,
+			className,
+			style,
+			selectedIndices,
+			expandedIndices,
+			...passThroughs
+		} = this.props;
+
+		const itemChildProps = VerticalListMenu.Item.findInAllAsProps(this.props);
+
+		return (
+			<ul
+				{...passThroughs}
+				className={boundClassNames('&', className)}
+				style={style}
+			>
+				{_.map(itemChildProps, (itemChildProp, index) => {
+					const {
+						hasExpander = false,
+					} = itemChildProp;
+
+					const itemChildrenAsArray = React.Children.toArray(itemChildProp.children);
+
+					// Was not able to get `child.Type` to work correctly, I suspect this
+					// is due to the way we wrap components with createLucidComponentDefinition
+					const listChildren = _.filter(itemChildrenAsArray, (child) => _.get(child, 'type.displayName', '') === 'VerticalListMenu');
+					const otherChildren = _.filter(itemChildrenAsArray, (child) => _.get(child, 'type.displayName', '') !== 'VerticalListMenu');
+
+					// If the prop is found on the child, it should override what was
+					// passed in at the top level for selectedIndices and expandedIndices
+					const actualIsExpanded = _.has(itemChildProp, 'isExpanded')
+						? _.get(itemChildProp, 'isExpanded', true)
+						: _.includes(expandedIndices, index);
+
+					const actualIsSelected = _.has(itemChildProp, 'isSelected')
+						? _.get(itemChildProp, 'isSelected', false)
+						: _.includes(selectedIndices, index);
+
+					return (
+						<li
+							key={index}
+							{...itemChildProp.passThroughs}
+							className={boundClassNames('&-Item', itemChildProp.className)}
+						>
+							<div
+								className={boundClassNames('&-Item-content', {
+									'&-Item-content-is-selected': actualIsSelected,
+								})}
+								onClick={_.partial(this.handleClickItem, index, itemChildProp)}
```

Comment:
Now that we're passing props through to event handlers is there a way to avoid using `partial`?